docs/astro.config.mjs

```diff
@@ -29,8 +10,11 @@ export default defineConfig({
       title: 'Open Payments',
       description:
         'An API for open access to financial accounts to send and receive payments.',
+      components: {
+        Header: './src/components/Header.astro'
+      },
       customCss: [
-        './node_modules/@interledger/docs-design-system/src/styles/green-theme.css',
+        './node_modules/@interledger/docs-design-system/src/styles/teal-theme.css',
```

Comment:
The only thing I would flag here is, is switching to the teal theme responsible for causing the darker grey code blocks in the API references?

![Screenshot 2024-02-19 at 15 33 38](https://github.com/interledger/open-payments/assets/15069181/34dc62e5-fe9c-46af-a020-9d144ba7dbfd)

Reply:
no, as part of the plugin update, it switched over to expressive code, and there are some style settings that will need to be over-ridden. I might escalate those into the shared design system, especially if Rafiki also wants to display OpenAPI docs.